client/web/compose/src/components/ModuleFields/Configurator/Number.vue

```diff
@@ -21,7 +22,7 @@
         cols="12"
         sm="6"
       >
-        <label class="d-block mb-3">
+        <label class="d-block mb-3 text-primary">
```

Comment:
Could you turn this into a b-form-group with a <template #label> for consistency.